server/config/environments.ts

```diff
@@ -0,0 +1,19 @@
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const {
+  NODE_ENV: nodeEnv,
+  DATABASE_DEV: dbDev,
+  DATABASE_TEST: dbTest,
+  DATABASE_URL: dbProduction,
+  SECRET_KEY: secretKey,
+} = process.env;
+
+export {
+  nodeEnv,
+  dbDev,
+  dbTest,
+  dbProduction,
+  secretKey,
+};
```

Comment:
export these as a single object 
`export const baseConfig = {}`
easier to read to know that its a configuration for example